src/MDMTransitionContext.h

```diff
@@ -80,3 +80,9 @@ NS_SWIFT_NAME(TransitionContext)
 @property(nonatomic, strong, readonly, nonnull) UIView *containerView;
 
 @end
+
+NS_SWIFT_NAME(InteractiveTransitionContext)
+@protocol MDMInteractiveTransitionContext
```

Comment:
Sounds like you want interactive transition and transition merged. So, I'll check if ```isInteractive``` is implemented. If it's not implemented, I assume it's not interactive, if it is implemented, I'll call ```isInteractive``` to determine if the direction (forward/backward) should be interactive.